content/docs/04-clusters/01-public-cloud/03-azure/01-azure-cloud.md

```diff
@@ -0,0 +1,127 @@
+---
+title: "Azure Cloud Account Creation"
+metaTitle: "Register Azure cloud account to Palette console"
+metaDescription: "The methods of registering Azure cloud account to Palette console"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Creating an Azure Cloud Account
```

Comment:
```suggestion
# Register an Azure Cloud Account
```
We avoid starting headlines with -ing. Here is the link to the section of the [style guide](https://spectrocloud.atlassian.net/wiki/spaces/DE/pages/1765933057/Spectro+Cloud+Internal+Style+Guide#Headings).